src/content/en/tools/service-worker-libraries/index.md

```diff
@@ -3,30 +3,64 @@ book_path: /web/tools/_book.yaml
 description: Service Worker Libraries.
 
 {# wf_published_on: 2015-01-01 #}
-{# wf_updated_on: 2016-09-12 #}
+{# wf_updated_on: 2016-11-03 #}
 
 # Service Worker Libraries {: .page-title }
 
-## What are the Service Worker Libraries?
+<div class="attempt-right">
+  <figure>
+    <img src="/web/tools/images/tools-landing-page.gif" width="80%">
```

Comment:
Remove all widths! `attempt-right` is designed to be responsive, if you add a size, you're breaking the point of that.